compiler/rustc_ast_lowering/src/expr.rs

```diff
@@ -41,8 +41,7 @@ impl<'hir> LoweringContext<'_, 'hir> {
                     }
                     // Merge attributes into the inner expression.
                     if !e.attrs.is_empty() {
-                        let old_attrs =
-                            self.attrs.get(&ex.hir_id.local_id).map(|la| *la).unwrap_or(&[]);
+                        let old_attrs = self.attrs.get(&ex.hir_id.local_id).copied().unwrap_or(&[]);
```

Comment:
```suggestion
                        let old_attrs = self.attrs.get(&ex.hir_id.local_id).copied().unwrap_or_default();
```

Reply:
I don't feel strongly about it, but very slightly prefer `unwrap_or`: it's both fewer keystrokes and slightly less cognitive load for the reader (no need to consider the type or what its `Default` implementation is).

Reply:
Both are present in the codebase, I don't think this should be changed, but if anything I think it should be in a follow-up PR.